summingbird-online-perf/src/main/scala/com/twitter/summingbird/online/executor/TestUtils.scala

```diff
@@ -0,0 +1,43 @@
+package com.twitter.summingbird.online.executor
+
+import org.openjdk.jmh.infra.Blackhole
+import scala.concurrent.duration.DurationInt
+import scala.util.{Failure, Success}
+
+object TestUtils {
+  val TickDelay = 5.millis
+
+  def testOperationContainer[I, O](
+    container: OperationContainer[I, O, InputState[Int]],
+    inputs: Array[I],
+    bh: Blackhole
+  ): Unit = {
+    var left = inputs.length
+    for (
+      input <- inputs;
+      emitted <- container.execute(InputState[Int](1), input)
```

Comment:
Do you have a sense of how expensive `container.execute` is? I ask because `testOperationContainer` introduces some noise so its overhead could dominate the final result.